src/routes/docs/products/auth/phone-sms/+page.markdoc

```diff
@@ -98,7 +98,7 @@ mutation {
 
 # Login {% #login %}
 
-After initiating the phone authentication process, the returned user ID and secret are used to confirm the user. The secret will usually be a 6-digit number in the SMS message sent to the user.
+After initiating the phone authentication process, the returned user ID and secret are used to confirm the user. The secret will usually be a 6-digit number in the SMS message sent to the user. You can also [mock the phone number](/docs/products/auth/security#mock-phone-number) to initiate a phone authentication process.
```

Comment:
This is a little hidden. Let's put a paragraph before the "Send SMS message' heading.

Also, we're not mocking phone numbers. "Mock" is an adjective.

Reply:
But the Console in the preview instance currently words it as Mock Phone Numbers, using "Mock" as a verb. Shouldn't we stay consistent with the Console? Or is this a mistake in the Console?

Reply:
> Console in the preview instance

This?

<img width="1208" alt="image" src="https://github.com/user-attachments/assets/e0cb79c2-677d-4660-bbf0-e86fd53bac2c">

Mock can either be a verb or adjective. In this case, it's an adjective.